<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>滤波反投影程序</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>1080</width>
      <height>360</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>图像显示</string>
    </property>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>50</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>原始图像</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>投影图像</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>310</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>反投影结果图像</string>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>label1</zorder>
    <zorder>label2</zorder>
    <zorder>label3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>1081</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Shepp-Logan模型</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="currentText">
      <string>128</string>
     </property>
     <item>
      <property name="text">
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Button3">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>滤波反投影</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>解析法投影</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>生成默认模型</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>650</y>
      <width>1081</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>一般图像处理</string>
    </property>
    <widget class="QPushButton" name="Button4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Radon变换投影</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button6">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>滤波反投影</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
